RecoHI/HiTracking/python/hiMergedConformalPixelTracking_cff.py

```diff
@@ -88,9 +88,10 @@
     makeReKeyedSeeds = cms.untracked.bool(False)
     )
 
-hiMergedConformalPixelTracking = cms.Sequence(
-    hiConformalPixelTracksSequence
-    *hiPixelOnlyStepSelector
-    *hiHighPtStepSelector
-    *hiGeneralAndPixelTracks
+hiMergedConformalPixelTrackingTask = cms.Task(
+    hiConformalPixelTracksTask
+    ,hiPixelOnlyStepSelector
+    ,hiHighPtStepSelector
+    ,hiGeneralAndPixelTracks
     )
+hiMergedConformalPixelTracking = cms.Sequence(hiMergedConformalPixelTrackingTask)
```

Comment:
This sequence is only used in RecoHI/HiTracking/python/HiTracking_cff.py and can be removed now